src/game/server/gamecontext.cpp

```diff
@@ -4478,3 +4496,30 @@ bool CGameContext::IsSpectatorCID(int ClientID) {
 	auto& vec = Server()->spectators_id;
 	return std::find(vec.begin(), vec.end(), ClientID) != vec.end();
 }
+
+void CGameContext::Converse(int ClientID, char *pStr)
```

Comment:
We have a player-used command `/mute`. It was added in c6a5028 and i have later added it to /msg (5c603c6). `/c` should honor it